SPECS/xapi.spec

```diff
@@ -1266,6 +1267,7 @@ done
 /opt/xensource/libexec/fence.bin
 /opt/xensource/debug/suspend-image-viewer
 %{_bindir}/list_domains
+%dir /etc/xenopsd-xc.conf.d
```

Comment:
So the directory is called `/etc/xenopsd-xc.conf.d`. Not completly consistent name `/etc/xenopsd.conf`, to me, but if that's what `xenopsd` currently wants, let it be.

Reply:
this is because the configuration file is changed in `xenopsd-xc.service` with the command line parameter: 
```
--config /etc/xenopsd.conf 
```
we could add
```
--config-dir /etc/xenopsd.conf.d
```
but this would move us away from upstream even more

Reply:
in that case, I'll suggest the changes in this PR to XS folks first to see if there are any disagreements

Reply:
we agreed to name the directory `xenopsd.conf.d` and the rest of the changes (though they are not going to move to `config` from `config(noreplace)`, which is fine)